index.d.ts

```diff
@@ -133,6 +133,38 @@ type ReaderRevenuePosition =
     | 'ampHeader'
     | 'ampFooter';
 
+interface CampaignsType {
+    id: string;
+    name: string;
+    rules: [];
```

Comment:
What is rule? Should this be `rules:RuleType[];`?

Reply:
Good point. Honestly I am not sure, the only example I have found has had an empty array. I struggle to find this value being used in frontend. Will investigate further

Reply:
I think it's better to only type and include the data we use. There's no reason the DCR types need to match those from frontend or even the data sent from frontend. Our types should document the data needed by the rendering tier.